test/lit/passes/O1_skip_passes.wast

```diff
@@ -0,0 +1,67 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+
+;; RUN: foreach %s %t wasm-opt -O2 --skip-pass=coalesce-locals --skip-pass=simplify-locals --skip-pass=simplify-locals-nostructure -S -o - 2>&1 | filecheck %s
+
+;; Check that we can skip several passes. Note that no local.tee is introduced.
+
+;; There should also be no warning in the output.
+
```

Comment:
```suggestion
```
May as well delete this as the test is not verifying that atm. (It could in theory if we added additional stuff here, but I don't think it's worth it.)